installer_scripts/install-yb-voyager

```diff
@@ -276,11 +286,12 @@ check_java() {
 	fi
 
     MIN_REQUIRED_MAJOR_VERSION='17'
+	MAX_REQUIRED_MAJOR_VERSION='19'
```

Comment:
let's add a comment here? Explaining that debezium build fails with 21 and above ..